<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFund_Voting</class>
 <widget class="QDialog" name="CFund_Voting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="windowMainTitle">
       <property name="font">
        <font>
         <family>.PingFang TC</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Proposal Voting</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>No vote selected:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="notvotingList">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Voting for:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="votingyesList">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Voting against:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="votingnoList">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="viewdetailsBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>View Proposal Details</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopvotingBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Voting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="voteyesBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Vote Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="votenoBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Vote No</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="switchBtn">
       <property name="text">
        <string>Switch to Payment Request View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="otherViewLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
